rules/windows/credential_access_iis_connectionstrings_dumping.toml

```
@@ -0,0 +1,48 @@
+[metadata]
+creation_date = "2020/08/18"
+ecs_version = ["1.5.0"]
+maturity = "production"
+updated_date = "2020/08/18"
+
+[rule]
+author = ["Elastic"]
+description = """
+An attacker with Microsoft IIS webserver access via a webshell or alike can decrypt and dump any hardcoded
+connectionstrings crednetials such as MSSQL service account password using aspnet_regiis command. Verify context of this
+execution such as parent process, file creation, username to validate the legitimacy of this action.
+"""
+index = ["winlogbeat-*", "logs-endpoint.events.*"]
+language = "kuery"
+license = "Elastic License"
+max_signals = 33
+name = "Microsoft IIS Connection Strings Decryption"
+references = [
+    "https://blog.netspi.com/decrypting-iis-passwords-to-break-out-of-the-dmz-part-1/",
+    "https://symantec-enterprise-blogs.security.com/blogs/threat-intelligence/greenbug-espionage-telco-south-asia",
+]
+risk_score = 73
+rule_id = "c25e9c87-95e1-4368-bfab-9fd34cf867ec"
+severity = "high"
+tags = ["Elastic", "Windows"]
+type = "query"
+
+query = '''
+event.category:process and not event.type:end and
```

```suggestion
event.category:process and event.type:(start or process_started) and
```

for process events, I would be explicit be using "start" because there is also an event.type of info which may result in unrelated or duplicative alerts. process_started is for backwards compat for pre 7.8 beats